k8s-daemonset/k8s/linkerd-zipkin.yml

```diff
@@ -54,6 +54,13 @@ data:
       servers:
       - port: 4141
         ip: 0.0.0.0
+
+    usage:
+      orgId: linkerd-examples-daemonset-zipkin
+
+    telemetry:
+    - kind: io.l5d.recentRequests
+      sampleRate: 0.25
```

Comment:
This config has 2 telemetry sections now, unfortunately. There's one defined on line 18, and again on line 61. Can you combine them? Also, maybe we should think about having linkerd abort if a config section shows up twice.